### DISCUSSION TITLE
Scheduled task will not complete overnight and throws error about screen unavilable

### DISCUSSION THREAD

**Discussion by hazydecisions** [d4rken-org/sdmaid-se#1400] [Q&A] [answered]
I also am not having my scheduled tasks run. I am getting a error:
Some scheduled tasks failed or encountered errors. Try running them manually. (and that works)
Appcleaner: eu.darken.sdmse.automation.core.errors.ScreenUnavailableException: Screen is unavailable!
The scheduled task has not ran overnight for some time.

**To Reproduce**
Overnight scheduled task run while plugged in. Phone is in "Bedtime mode" during this time, and is plugged in.

**Expected behavior**
Task completes without manual intervention

**Smartphone (please complete the following information):**
SD-Maid SE current version v1.2.5-rc0
Google Pixel 8 Pro
Android 14
Patched: 5 September 2024

**Comment by MarkyMarkDE** [comment, marked as answer]
I bet, you have the accessibility mode (ACS) active for your scheduled task, right?
See:
![Screenshot_20240911-031257](https://github.com/user-attachments/assets/f5080b3a-61a2-410d-b14e-2a783c1f9f2c)
Then it seems a normal behavior, all actions with the accessibility mode requires(!) a unlocked and active screen. 
Try the same without accessibility mode in scheduled task.

If this is not given in your case please provide a debug log, so the Dev can check it.
Please see [FAQ reporting an issue / debug log]( https://github.com/d4rken-org/sdmaid-se/wiki/Bugs). 

The community behind of SD-Maid SE thanks for your help in advance!

**Comment by hazydecisions** [comment]
I disabled this and the task did run, but I needed to click the delete button manually. I know in previous versions that it just "did its thing" overnight and I woke up to clean device. However if additional security is now in place that prevents this from happening I guess that I can accept that this is the case. 

If it is how Android now works I will  use the application without a scheduler. Thank you for the clarification.

**Reply by MarkyMarkDE** [reply]
> I disabled this and the task did run, but I needed to click the delete button manually. I know in previous versions that it just "did its thing" overnight and I woke up to clean device. However if additional security is now in place that prevents this from happening I guess that I can accept that this is the case.
> 
> If it is how Android now works I will use the application without a scheduler. Thank you for the clarification.

yes, unfortunately the Rules from G* for usage of some functions e.g. the ACS is going more and more strict from API level to API level.
I would also wish it would be more lax ...
